rdf/rdf12/rdf-turtle/syntax/turtle-star-annotation-6.ttl

```diff
@@ -0,0 +1,3 @@
+PREFIX : <http://example/>
+
+:s :p :o ~ .
```

Comment:
Its' a corner-case, as the production for xxx as the production is:

```
reifier               ::= '~' (iri | BlankNode)?
```

Otherwise, `:s :p :o ~` is equivalent to `:s :p :o ~ []` or `:s :p :o {||}` (that last is in the SPARQL grammar, but not the Turtle grammar.